docs/concepts/transaction.md

```diff
@@ -50,7 +50,108 @@ There are several ways of creating `Receipts`:
+    outcome: {
+      logs: [ [length]: 0 ],
+      receipt_ids: [ '6LrHPazG3DTcKkd4TjqbgajqmbcAfyoTG383Cft5SZ5Y', [length]: 1 ],
+      gas_burnt: 223182562500,
+      tokens_burnt: '22318256250000000000',
+      executor_id: 'near',
+      status: {
+        SuccessReceiptId: '6LrHPazG3DTcKkd4TjqbgajqmbcAfyoTG383Cft5SZ5Y'
+      }
+    }
+  },
+  receipts_outcome: [
+    {
+      proof: [ [length]: 0 ],
+      block_hash: '6evPKFQRw1E3gH9L1d59mz7GahsbnqsdYwcZQo8hpFQB',
+      id: '6LrHPazG3DTcKkd4TjqbgajqmbcAfyoTG383Cft5SZ5Y',
+      outcome: {
+        logs: [ [length]: 0 ],
+        receipt_ids: [ '7NMpF9ZGwSj48bpvJK2xVobJkTasEkakazTKi2zotHR4', [length]: 1 ],
+        gas_burnt: 223182562500,
+        tokens_burnt: '22318256250000000000',
+        executor_id: 'transfer-vote.near',
+        status: { SuccessValue: '' }
+      }
+    },
+    {
+      proof: [ [length]: 0 ],
+      block_hash: 'Gm6TFS1ZxmA45itVj8a7vE8yJF8V5hXeNF1EhEVr7GVS',
+      id: '7NMpF9ZGwSj48bpvJK2xVobJkTasEkakazTKi2zotHR4',
+      outcome: {
+        logs: [ [length]: 0 ],
+        receipt_ids: [ [length]: 0 ],
+        gas_burnt: 0,
+        tokens_burnt: '0',
+        executor_id: 'near',
+        status: { SuccessValue: '' }
+      }
+    },
+    [length]: 2
+  ]
+}
+```
+where it displays the overall status of the transaction, the outcomes of the transaction, and the outcome of the receipts
+generated by this transaction. 
+
+The `status` field is an object with a single key. There can be four types of key:
+- `SucccessValue`.  This indicates that the receipt has been successfully executed and the value of the key
+is the return value (can only be nonempty when it is the result of a function call receipt).
+- `SuccessReceiptId`. This indicates that either a transaction has been succesfully converted to a receipt or a receipt is
+successfully processed and generated another receipt. The value of this key is the id of the newly generated receipt.
+- `Failure` This indicates that the transaction or receipt has failed.
+- `Unknown` This indicates that the transaction or receipt hasn't been processed yet.
+
+The top level `status` indicates the overall status of the transaction. For transaction that does not include function
+call actions, this means whether all the actions are successfully executed. For function call transaction, this indicates
+whether the function call has been successfully executed. However, one caveat is that the successful execution of the
+function call does not necessarily mean that the receipts spawned from the function call are all successfully executed.
+For example, 
+```rust
+pub fn transfer(receiver_id: String) {
+    Promise::new(receiver_id).transfer(10);
+}
+```
+this function schedules a promise, but its return value is unrelated to the promise that it has scheduled. So even if the
+promise fails, potentially due to a lack of attached deposit, a transaction that calls this function will still have
```

Comment:
> potentially due to a lack of attached deposit

This will fail at the outgoing receipt validation, so the TX will fail.

Better to say if the `receiver_id` doesn't exist